### PR TITLE
Frontend for chat-feature

### PR DESCRIPTION
**Pull request by Itsmeeeet**
### **Frontend for the chat-feature**

- chatbox (on the left)
- my contacts (on the right)

_Frontend tests still remaining_

### CODE REVIEW

**Review by DasDeveloper — COMMENTED** [review]
The page is not responsive.

File: src/client/src/components/Chat.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            <l1 className="connectionsInfo">
+              <img
+                src="https://i.pinimg.com/originals/f1/0f/f7/f10ff70a7155e5ab666bcdd1b45b726d.jpg"
+                alt="comapnyPic"
+                className="companyPic"
+              ></img>
+              <div>
+                <span className="connectionName">Jack Billionaire</span>
+              </div>
+            </l1>
```
Comment (DasDeveloper) [inline src/client/src/components/Chat.jsx:67]: The contacts should be centered. 
Screen Shot 2023-03-09 at 11 17 09 AM

File: src/client/src/components/Chat.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        </div>
+        <div class="vl"></div>
+        <div className="sender-message">
+          <div className="sender-userpic"></div>
+          <div className="sender-text">Hi, how are you?</div>
+        </div>
+        <div className="user-message">
+          <div className="user-text">Hi, I am doing great! What about you?</div>
+          <div className="user-userpic"></div>
+        </div>
```
Comment (DasDeveloper) [inline src/client/src/components/Chat.jsx:23]: This div is on top of the message input. You need to add a margin.
Screen Shot 2023-03-09 at 11 17 09 AM

**Review by DasDeveloper — COMMENTED** [review]
The page is also a bit too big, please make it a bit smaller.

File: src/client/src/components/Chat.jsx (hunk 1 above)
Comment (Kunal22shah) [reply src/client/src/components/Chat.jsx:67]: image
The css is messed up on my end as well

**Review by Kunal22shah — CHANGES_REQUESTED** [review]
The css alignments seem off, other that that looks good!

**Review by DasDeveloper — CHANGES_REQUESTED** [review]
Frontend tests missing for this page

**Pull request closed without merging**